docs/flow/concepts/configuration_language.md

```diff
@@ -6,3 +6,86 @@ weight: 300
 ---
 
 # Configuration language
+
+The Grafana Agent Flow _configuration language_ refers to the language used in
+configuration files which define and configure components to run.
+
+The configuration language is called River, a Terraform/HCL-inspired language:
+
+```river
+metrics.scrape "default" {
+  targets = [{
+    "__address__" = "demo.robustperception.io:9090",
+  }]
+  forward_to = [metrics.remote_write.default.receiver]
+}
+
+metrics.remote_write "default" {
+  remote_write {
+    url = "http://localhost:9009/api/prom/push"
+  }
+}
+```
+
+River was designed with two requirements in mind:
+
+* _Fast_: The configuration language must be fast so the component controller
```

Comment:
This could trigger discussions if River is faster than HCL if we don't detail why is fast. Probably same for the rest of requirements.

Reply:
Agreed, but I think the detailed docs for River are a better place to talk more about the speed rather than the high-level overview page this acts as.